src/plugins/git/explorerAdapter.js

```diff
@@ -44,6 +36,7 @@ export class StaticExplorerAdapter implements IStaticExplorerAdapter {
     const [graph, repository] = await Promise.all([
```

Comment:
we can remove the `Promise.all` since now there's only one promise.

`const repository = await loadRepository()`